packages/lib/services/plugins/defaultPlugins/desktopDefaultPluginsInfo.ts

```diff
@@ -0,0 +1,16 @@
+import { InitialSettings } from '../PluginService';
+import Setting from '../../../models/Setting';
+
+export const defaultPlugins = {
+	'io.github.jackgruber.backup': '1.0.2',
+	'plugin.calebjohn.rich-markdown': '0.8.3',
+};
+
+export function getInitialSettings(): InitialSettings {
+	const initialSettings: InitialSettings = {
+		'io.github.jackgruber.backup': {
+			'path': `${Setting.value('profileDir')}`,
+		},
+	};
+	return initialSettings;
```

Comment:
Try to have a single object with all plugin related properties in there. For example:

```typescript
export const defaultPlugins = {
	'io.github.jackgruber.backup': {
		version: '1.0.2',
		settings: {
			'path': `${Setting.value('profileDir')}`,
		},
	'plugin.calebjohn.rich-markdown': {
		version: '0.8.3',
	},	
};
```

That way you have everything in one place and don't need to repeat the key for each piece of info you want to add.

Reply:
This way it is a lot cleaner.